pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -482,6 +482,54 @@ func aclToRules(aclArgs types.AppNetworkACLArgs, ACLs []types.ACE) (types.IPTabl
 		aclRule5.Action = []string{"-j", chainName}
 		aclRule5.ActionChainName = chainName
 		rulesList = append(rulesList, aclRule5)
+	} else {
```

Comment:
Else case will be executed when IPVer = 6. Is that what you intended?

Reply:
No that wasn't my intention. I'll fix this.

Reply:
I see we only handle the switch case for ipVer==4, which is a bit odd. But I'll let it slide.
We need to get all of our test cases to use IPv6 and then we can revisit this.